### ISSUE TITLE
Formatting error with is a mix of #uuid and comment

### ISSUE THREAD

**Issue by chpill** [oakmac/standard-clojure-style-js#165]
Hello, thank you for this library!

I have encountered an issue formatting a codebase at work (using `v0.16.0`). I have tried narrowing down the problem, consider a file containing only the following form:

```clojure
(case
  #uuid "96bbb029-40e7-4566-83d5-7e6088a908fc" :a
  ;; a comment
  #uuid "148ccf8b-6122-4435-9a39-001e13a8aad7" :b)
```

If I call `fix` on it, standard-clj will add a space in front of the comment (from my understanding of the formatting rules, it shouldn't).

```clojure
(case
  #uuid "96bbb029-40e7-4566-83d5-7e6088a908fc" :a
   ;; a comment
  #uuid "148ccf8b-6122-4435-9a39-001e13a8aad7" :b)
```

Then, if I call `fix` again, it removes the space it just added, and this goes on forever.

Calling `check` on either of the forms above will report "file require formatting", which is consistent at least.

For those that encounter this issue, moving the comment elsewhere will prevent this weird behavior.

**Comment by oakmac** [comment]
Thank you for the report! This looks like a bug to me.

**Comment by oakmac** [comment]
Fixed with [PR-168](https://github.com/oakmac/standard-clojure-style-js/pull/168) and available with v0.18.0. Can you check to see if this fixes the issue with your codebase @chpill?

**Comment by chpill** [comment]
@oakmac I just upgraded to v0.18.0 and it fixes the issue. Thank you!
